templates/WinUI/Projects/Default/.template.config/template.json

```diff
@@ -84,8 +84,8 @@
       "manualInstructions": [ ],
       "actionId": "0B814718-16A3-4F7F-89F1-69C0F9170EAD",
       "args": {
-        "packageId" : "Microsoft.Xaml.Behaviors.WinUI.Managed",
-        "version" : "2.0.3-rc6",
+        "packageId" : "CommunityToolkit.WinUI.UI.Behaviors",
```

Comment:
We should keep Microsoft.Xaml.Behaviors.WinUI.Managed, but update to rc7, CommunityToolkit.WinUI.UI.Behaviors is built on top of it + pulls it in as a dependency.